src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -18,24 +20,39 @@ public class Limelight extends SubsystemBase {
   // private double[] targetPose = null;
   private Pose3d botPose;
 
+  private final InterpolatingDoubleTreeMap shooterMap;
```

Comment:
Use `InterpolatingDoubleTreeMap<Double, Double>` as the type. This makes it clear that the Tree map maps a `Double` to a `Double`.

Reply:
this wpilib class doesn't accept type parameters

Reply:
Sorry, I didn't realize InterpolatingDoubleTreeMap extended the InterpolatingTreeMap<Double, Double> class. Resolved.